cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -793,12 +796,30 @@ def test_service_service_vms_retires_archived():
     pass
 
 
-@pytest.mark.meta(coverage=[1558926])
-@pytest.mark.manual
+@pytest.fixture(scope='function')
+def full_vm(appliance, provider, full_template):
```

Comment:
Question:
Why do create/write `full_vm` function scoped fixture, as we already have one .?

Reply:
Thanks for this suggestion.